common/djangoapps/edxmako/paths.py

```diff
@@ -49,15 +54,23 @@ def add_directory(self, directory, prepend=False):
 
     def get_template(self, uri):
         """
-        Overridden method which will hand-off the template lookup to the microsite subsystem
+        Overridden method do template lookup database or in white labe site theme.
+        If not found template lookup will be done in comprehensive theme for current site
+        by prefixing path to theme.
+        e.g if uri is `main.html` then new uri would be something like this `/red-theme/lms/static/main.html`
+        If still unable to find template it fallback to default template dirs after striping off the
```

Comment:
"If still unable to find a template, it will fallback to the default template directories after stripping off the prefix..."